docs/develop/01_sdk/03_chain_setup/01_standalone_setup.md

```diff
@@ -12,7 +12,7 @@ This greatly simplifies the setup.
 You only need to start the Docker image:
 
 ```bash
-docker run --rm -it -p 9944:9944 -p 9933:9933 kiltprotocol/mashnet-node:latest --dev --ws-external --rpc-external
+docker run --rm -it -p 9944:9944 -p 9933:9933 kiltprotocol/mashnet-node:latest --dev --rpc-external
```

Comment:
As far as I know only port 9944 is exposed now, right? The old WS port is not exposed anymore, so the port publishing should only have 9944 exposed.

Reply:
Or if only 9933 is used, 9944 is outdated.